### DISCUSSION TITLE
3.9.0 版本升级后启动项目报错

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#234] [02.提问]
![0610D0745151C4168E656484BE3E8320](https://user-images.githubusercontent.com/6092525/206337119-2264ffb0-c563-4d01-938b-43bdb4285911.jpg)

**Comment by zuihou** [comment]
注意对比lamp-util 改动， 涉及到 src/main/resources/ 的改动

**Reply by zqb666** [reply]
能不能麻烦告诉我一下 这个错误 是怎么引起的 我对了一上午 应该已经把这次更新的内容同步了 gateway服务 启动还是报这个错

**Comment by zuihou** [comment]
我还没强到一个截图就能判断报错是怎么引起的

**Comment by zuihou** [comment]
gateway 里面搜  lbResetTemplate , lamp-cloud-starter 里面搜  lbResetTemplate 

gateway 里面注入了 lbResetTemplate ， 但服务启动时，没有实例 lbResetTemplate。

在lamp-cloud-starter 里面搜 lbResetTemplate ，然后断点，启动时，需要先执行  @Bean lbResetTemplate 这个，才能启动成功

**Reply by zqb666** [reply]
没事了 lamp-cloud-starter resource目录META-INF 下面 没有建 spring文件夹 引起的
